cv.md

```
@@ -0,0 +1,83 @@
+- Total Rewards analytics and storytelling
+- Executive presentations and C-suite communication
+- Benefits and HR program optimization
+- Cross-functional project and program management
+- Dashboard development with Power BI and Tableau
+- Process design, automation, and workflow improvement
+- SQL, Excel, PowerQuery, and data analysis
+- Stakeholder engagement, vendor partnership, and change leadership
+
+## Professional Experience
+
+### Wellstar Health System, Atlanta, GA
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)